man/resolved.conf.xml

```diff
@@ -213,6 +213,18 @@
         (such as 127.0.0.1 or ::1), in order to avoid duplicate local caching.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStub=</varname></term>
+        <listitem><para>Takes a boolean argument or one of <literal>udp-only</literal> and <literal>tcp-only</literal>.
```

Comment:
Please name the options "udp" and "tcp". For example, we have `DHCP=ipv4` and `DHCP=ipv6`, w/o "-only", so it's better to follow convention, and also simpler is better.

I'd also rename the option to DNSStubListener=, after all resolved as a whole is a stub resolver, so "DSNStub" is a bit confusing. @poettering?